### PR TITLE
Close issue #148.

### PR DESCRIPTION
**Pull request by favonia**

### CODE REVIEW

File: library/paths/s1.red [hunk 1]
```diff
@@ -107,7 +107,7 @@ def decode : (x : s1) → s1-univ-cover x → path s1 base x =
   elim [
   | base → loopn
   | loop i → λ y j →
-    let n : int = ua/proj int int isuc/equiv i y in
+    let n : int = y .vproj in
```
Comment (jonsterling) [inline library/paths/s1.red]: whoa this is sweet.

File: library/prelude/univalence.red [hunk 2]
```diff
@@ -7,28 +7,22 @@ def ua (A B : type) (E : equiv A B) : path^1 type A B =
...
         let ctr/B (j : 𝕀) : B =
           comp 1 j b [
-          | i=0 → w.snd.snd b .fst .snd
+          | i=0 → w .snd .snd b .fst .snd
           | i=1 → refl
           ]
         in
         let ctr : fiber VB B proj/B b =
-          (`(vin i (fst (fst ((snd (snd w)) b))) (@ ctr/B 0)), ctr/B)
+          ((w .snd .snd b .fst .fst, ctr/B 0), ctr/B)
```
Comment (jonsterling) [inline library/prelude/univalence.red]: sweet pair notation :sunglasses:

File: library/prelude/univalence.red [hunk 3]
```diff
@@ -40,6 +34,7 @@ def univalence (B : type) : is-contr^1 ((A : type) × equiv A B) =
             | j=1 → ctr/B
             ]
           in
+          -- MORTAL this should be: w.snd.snd b .snd v j .fst
           ( `(vin i (fst (@ ((snd ((snd (snd w)) b)) v) j)) (@ filler 0))
```
Comment (jonsterling) [inline library/prelude/univalence.red]: oh this is the part that needed help lol

**Comment by jonsterling** [comment]
I'm ok with merging this, since the one part that doesn't work is a bug in the unifier that we can fix separately.

File: src/core/Tm.ml [hunk 4]
```diff
@@ -1081,8 +1081,8 @@ and pp_frame env fmt =
     Format.fprintf fmt "car"
   | Snd ->
     Format.fprintf fmt "cdr"
-  | VProj _ ->
-    Format.fprintf fmt "<vproj>"
```
[the remark this answers is not in the document]
Comment (favonia) [reply src/core/Tm.ml]: :rofl:

[the remark this answers is not in the document]
Comment (jonsterling) [reply src/core/Tm.ml]: taste it!

[the remark this answers is not in the document]
Comment (jonsterling) [reply src/core/Tm.ml]: also, wtf are those still being printed as `car`?

**Pull request merged**